src/typescript/rollup-plugin-fable/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rollup-plugin-fable",
+  "version": "0.0.0",
+  "description": "Rollup Plugin for fable-compiler",
+  "main": "index.js",
+  "scripts": {
+    "eslint": "eslint ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fable-compiler/Fable.git"
```

Comment:
Sorry, I should have explained more.

The URL points to `Fable`, but that's not very helpful, since what the person is looking for is `rollup-plugin-fable`, so they have to dig through the folders to find it.

Babel also uses a monorepo, but their [npm packages point to the correct directory within the monorepo](https://www.npmjs.com/package/babel-preset-es2015).

Notice how [it links to `https://github.com/babel/babel/tree/master/packages/babel-preset-es2015`](https://github.com/babel/babel/blob/master/packages/babel-preset-es2015/package.json#L8) rather than `https://github.com/babel/babel/tree/master` or `https://github.com/babel/babel.git`